src/EnergyPlus/DataHeatBalFanSys.hh

```diff
@@ -100,15 +100,16 @@ namespace DataHeatBalFanSys {
 
 struct HeatBalFanSysData : BaseGlobalStruct
 {
-    Array1D<Real64> SumConvHTRadSys;         // Sum of convection to zone air from hi temp radiant heaters
-    Array1D<Real64> SumLatentHTRadSys;       // Sum of latent gains from hi temp radiant heaters
-    Array1D<Real64> SumConvPool;             // Sum of convection to zone air from pools
-    Array1D<Real64> SumLatentPool;           // Sum of latent gains from pools
-    Array1D<Real64> QHTRadSysToPerson;       // Sum of radiant gains to people from hi temp radiant heaters
-    Array1D<Real64> QHWBaseboardToPerson;    // Sum of radiant gains to people from hot water baseboard heaters
-    Array1D<Real64> QSteamBaseboardToPerson; // Sum of radiant gains to people from steam baseboard heaters
-    Array1D<Real64> QElecBaseboardToPerson;  // Sum of radiant gains to people from electric baseboard heaters
-    Array1D<Real64> QCoolingPanelToPerson;   // Sum of radiant losses to people from cooling panels
+    Array1D<Real64> SumConvHTRadSys;             // Sum of convection to zone air from hi temp radiant heaters
+    Array1D<Real64> SumLatentHTRadSys;           // Sum of latent gains from hi temp radiant heaters
+    Array1D<Real64> SumConvPool;                 // Sum of convection to zone air from pools
+    Array1D<Real64> SumLatentPool;               // Sum of latent gains from pools
+    Array1D<Real64> SurfQdotRadHVACToPerson;     // Sum of radiant gains to people from all radiant HVAC sources
```

Comment:
Lots of nice renaming in this PR, except the `*ToPerson` variables are not `Surf` values, they are `Zone` values.

Reply:
That makes sense, I'll make this switch.